client/src/pages/private/ParticipantTable.js

```diff
@@ -244,7 +245,31 @@ export default () => {
       });
     }
   };
-
+  const handleAcknowledge = async (id) => {
```

Comment:
For now this is Ok, but in future we should remove all api call logic to service layer. This file is also growing massively.

Reply:
Agreed!